src/main/java/com/example/Trello/controllers/CardController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.Trello.controllers;
+
+import com.example.Trello.model.dto.card.CardCreation;
+import com.example.Trello.model.entity.Card;
+import com.example.Trello.services.CardService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/boards/{board_id}")
```

Comment:
```suggestion
@RequestMapping("/boards/{board_id}/cards")
```
then you can remove `/cards` from all handlers